src/scripts/components/main/embed-wizard/embed-wizard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {FunctionComponent, useState} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import {embedElementsSelector} from '../../../selectors/embed-elements-selector';
+import {ElementOptions} from '../../../types/embed-elements';
+import EmbedCheckboxList from '../embed-checkbox-list/embed-checkbox-list';
+import EmbedResult from '../embed-result/embed-result';
+
+import styles from './embed-wizard.module.styl';
+
+const EmbedWizard: FunctionComponent = () => {
+  const embedElements = useSelector(embedElementsSelector);
+  const [appElementsChecked, setAppElementsChecked] = useState(
+    embedElements as ElementOptions
+  );
+
+  const disabledParamsString = Object.entries(appElementsChecked)
```

Comment:
Perhaps better to pass this not as a string but as the object and only transform (with URLSearchParams) when it gets rendered

Reply:
Ah of course, makes sense 👍🏼